Templates/CSharp/EdgeDriverTemplate/source.extension.vsixmanifest

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0" encoding="utf-8"?>
+<PackageManifest Version="2.0.0" xmlns="http://schemas.microsoft.com/developer/vsx-schema/2011" xmlns:d="http://schemas.microsoft.com/developer/vsx-schema-design/2011">
+    <Metadata>
+        <Identity Id="EdgeDriverTest.0b43ac6b-e6ff-4afc-a5f6-286d41c046a6" Version="15.0.27805.00" Language="en-US" Publisher="Microsoft" />
```

Comment:
You'd have to change this version to 16.0.28113.00 keep it consistent with other templates.

Reply:
OK, I will change it in next commit